java/src/jmri/jmrix/loconet/pr3/PR3SystemConnectionMemo.java

```diff
@@ -149,7 +149,7 @@ public boolean provides(Class<?> type) {
         if (type.equals(jmri.PowerManager.class)) {
             return true;
         }
-        return false;
+        return super.provides(type);
```

Comment:
@bobjacobsen same basic question as above, and for the same reasons.  Will this cause issues with PR3 connections?

Reply:
I think that the `super.provides(type)` is a good change, but perhaps I'm not understanding the question.

Reply:
The question here was is this change going to enable anything provided by the LoconetSystemConnectionMemo that isn't supposed to be enabled on the PR3

Reply:
Ah, thanks, understand.  In MS100 mode, it's OK to invoke `super`, but in standalone programmer `return false` is appropriate to avoid providing TurnoutManager et al, similar to PR2 comment above.